engine/src/main/java/org/mides/optimization/model/Problem.java

```diff
@@ -71,12 +98,12 @@ public void initialize() {
                 );
         }
 
-        numberOfNodes = rideRequests.size() * 2 + 1;
+        numberOfNodes = index; // rideRequests.size() * 2 + 1; // Updated for per-vehicle depots
```

Comment:
Esto no sería la cantidad de viajes (ida y vuelta)  + la cantidad de depositos?

```suggestion
        numberOfNodes = rideRequests.size() * 2 + index;
```

Reply:
si y esta contando todo, primero suma los depot en el for de arriba y despues los viajes. Lo que me hicidte dudar es si falta es numberofdepots= index +1 xq index arranca en 0.

Reply:
le sume 1

Reply:
En verdad arrancaba en 1 si